PROTO_tests/tests/alignment_test.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+
+import arkouda as ak
+
+DATA_TYPES = [ak.int64, ak.uint64, ak.float64]
+
+
+class TestAlignment:
+    @staticmethod
+    def get_interval_info(lower_bound, upper_bound, vals, dtype):
+        lb = ak.array(lower_bound, dtype)
+        ub = ak.array(upper_bound, dtype)
+        v = ak.array(vals, dtype)
+
+        if dtype == ak.uint64:
+            lb += 2 ** 63
+            ub += 2 ** 63
+            v += 2 ** 63
+        elif dtype == ak.float64:
+            lb += 0.5
+            ub += 0.5
+            v += 0.5
+        return lb, ub, v
+
+    @pytest.mark.parametrize("dtype", DATA_TYPES)
+    def test_search_interval(self, dtype):
+        expected_result = [2, 5, 4, 0, 3, 1, 4, -1, -1]
```

Comment:
I think we could also include this in the `get_interval_info` method.

Reply:
The expected_result values are different for the two calling functions.  I also think it may be more readable to have the expected_result listed with the bounds and values arrays?

Reply:
yah I think we leave `expected_result` out of the `get_interval_info` since it's a different value in `test_search_interval_nonunique` than here

Reply:
Yeah, y'all are correct. I was reading too quickly.